pkg/controllers/disruption/metrics.go

```diff
@@ -75,7 +75,17 @@ var (
 			Name:      "consolidation_timeouts_total",
 			Help:      "Number of times the Consolidation algorithm has reached a timeout. Labeled by consolidation type.",
 		},
-		[]string{consolidationTypeLabel},
+		[]string{ConsolidationTypeLabel},
+	)
+	InvalidatedConsolidationTotal = opmetrics.NewPrometheusCounter(
```

Comment:
Is it `failed_validations` or `invalidations`? I'm not sure that I have a strong opinion either way -- I do think that we need to make sure that the name of the actual go variable closely matches the naming in prometheus

Reply:
I think it's probably better put as `failed_validations`